src/reason-parser/reason_lexer.mll

```diff
@@ -636,6 +636,11 @@ rule token = parse
       }
 
 and enter_comment = parse
+  | "//" ([^'\010']* newline as line)
```

Comment:
@let-def Are there other kinds of newlines we need to watch out for (something like windows)?